### DISCUSSION TITLE
Using the BTS 7960 What Pins to Use

### DISCUSSION THREAD

**Discussion by SandokahnR5** [ClassicDIY/SkyeTracker#34] [Q&A] [answered]
Hello i am just in Assembling the whole thing and as in Germany the BTS 7960 Modules are quickly available i have bought these so my Question is how to wire them up because the have dedicated Pins for pwm Left and Right.
I havnt even startetd to assemble the whole Tracker so try to get answer for my question in the first place.
Thank you for your time

**Comment by graham22** [comment]
Assuming you're using the ESP32 version, you should connect GPIO13 to RPWM (Horizontal), GPIO12 to R_EN (Horizontal) and GPIO27 to LPWM (Vertical) , GPIO26 to L_EN (vertical)
[ESP32-wiring-diagram](https://github.com/ClassicDIY/SkyeTracker/wiki/3---ESP32-wiring-diagram)

**Reply by SandokahnR5** [reply]
Yes i am Using the ESP32 Version, i will Try it.
Its a bit confusing for me, i am better in using a Mill and a Lathe than Bits and Bytes .

**Comment by SandokahnR5** [comment, marked as answer]
![DSC_1085](https://user-images.githubusercontent.com/108401794/179522043-b5ad73ad-4dba-4083-96d7-4636c4cf66a1.JPG)
So far my Setup but i dont get any Movement, wich Connection have i to make?
Sorry for the mybe stupid Question

**Reply by graham22** [reply]
you could try wiring both R_EN and L_EN to GPIO_12 , GPIO_13 to RPWM and GPIO_14 to LPWM  for the horizontal actuator 
the vertical actuator would be R_EN and L_EN to GPIO_26, GPIO_27 to RPWM and GPIO_25 to LPWM.
You can find the code that interfaces with the controller in the LinearActuatorNoPot.cpp,
